pymc/sampling/mcmc.py

```diff
@@ -499,6 +504,13 @@ def sample(
         Which NUTS implementation to run. One of ["pymc", "nutpie", "blackjax", "numpyro"].
         This requires the chosen sampler to be installed.
         All samplers, except "pymc", require the full model to be continuous.
+    blas_cores: int or "auto" or None, default = "auto"
+        The total number of threads blas and openmp functions should use during sampling. If set to None,
```

Comment:
Explain the default first?

Reply:
+1
Do you think we should already default to "auto", or first release something where the default is None so that this can be tested a bit more?

Reply:
I think the new default makes much more sense. This often shows up in MvNormal models and it's very tricky for beginners to debug